tests/p2p/test_peer_id.py

```diff
@@ -188,7 +194,9 @@ def getPeer(self):
 
     def test_validate_certificate(self):
         peer = PeerId('testnet')
-        protocol = HathorProtocol('testnet', peer, None, node=None, use_ssl=True, inbound=True)
+        # TODO: also test with sync-v2 enabled
+        protocol = HathorProtocol('testnet', peer, None, node=None, use_ssl=True, inbound=True,
+                                  enable_sync_v1=True, enable_sync_v2=False)
```

Comment:
Should we use `self._enable_sync_v1` and `self._enable_sync_v2`?

Reply:
Fixed.